NOLF2/ClientShellDLL/ClientShellShared/GameClientShell.cpp

```diff
@@ -873,6 +873,8 @@ void CGameClientShell::CSPrint(const char* msg, ...)
 
 uint32 CGameClientShell::OnEngineInitialized(RMode *pMode, LTGUID *pAppGuid)
 {
+	SDL_Window* window;
+	CUserProfile* pProfile;
```

Comment:
just declare and initialize this on 1123, there's no need for it at the start of the function